src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/RetryPolicyUtils.cs

```diff
@@ -26,9 +30,25 @@ public static class RetryPolicyUtils
         /// Blocklist of non-transient errors that should stop retry during data transfer operations
         /// </summary>
         private static readonly HashSet<int> _nonRetryableDataTransferErrors;
+        /// <summary>
+        /// Max intervals between retries to wake up serverless instances.
+        /// </summary>
+        private const int _serverlessMaxIntervalTime = 30;
+        /// <summary>
+        /// Maximum number of retries to wake up serverless instances.
+        /// </summary>
+        private const int _serverlessMaxRetries = 4;
 
         static RetryPolicyUtils()
         {
+            _retryableExtendedNetworkConnectivityError = new HashSet<int>
```

Comment:
How did you come up with this name? I'm not really sure how it relates to the error inside. And given that it is only used for the "serverless" retry shouldn't it say something about that?